bazel/verilog.bzl

```diff
@@ -60,8 +60,12 @@ def _verilog_base_impl(ctx, subcmd, test = True, extra_args = [], extra_runfiles
     runfiles += ctx.files.verilog_runner_tool
     srcs = get_transitive(ctx = ctx, srcs_not_hdrs = True).to_list()
     hdrs = get_transitive(ctx = ctx, srcs_not_hdrs = False).to_list()
-    src_files = [src.short_path for src in srcs]
-    hdr_files = [hdr.short_path for hdr in hdrs]
+    if test:
+        src_files = [src.short_path for src in srcs]
+        hdr_files = [hdr.short_path for hdr in hdrs]
+    else:
+        src_files = [src.path for src in srcs]
+        hdr_files = [hdr.path for hdr in hdrs]
```

Comment:
What's the story with `path` vs `short_path`?

Reply:
build target uses `path`, test target uses `short_path` (not sure why bazel makes a difference between the two)